Enhancement: self defined `Box` type is conflict with `std::boxed::Box`

If protocol buffer files defined a type `Box`, the generated source file can't be compile for different types:

```
9188 |     fn into_any(self: Box<Self>) -> ::std::boxed::Box<dyn (::std::any::Any)> {
     |                           ^^^^ unexpected type argument

error[E0107]: wrong number of type arguments: expected 0, found 1
```

Ver: 2.14

--- a/protobuf-codegen/src/message.rs
+++ b/protobuf-codegen/src/message.rs
@@ -369,7 +369,7 @@ impl<'a> MessageGen<'a> {
                 w.write_line("self as &mut dyn (::std::any::Any)");
             });
             w.def_fn(
-                "into_any(self: Box<Self>) -> ::std::boxed::Box<dyn (::std::any::Any)>",
+                "into_any(self: ::std::boxed::Box<Self>) -> ::std::boxed::Box<dyn (::std::any::Any)>",
                 |w| {
                     w.write_line("self");
                 },

